framework/src/main/java/org/checkerframework/framework/type/AnnotationClassLoader.java

```diff
@@ -89,6 +93,7 @@
     protected final ProcessingEnvironment processingEnv;
 
     /** The resource URL of the qual directory of a checker class. */
+    @SuppressWarnings("UnusedVariable")
```

Comment:
This seems to still be used.  Is this an old `@SuppressWarnings` that should now be removed?